apps/server/src/modules/learnroom/uc/lesson-copy.uc.ts

```diff
@@ -55,10 +47,24 @@ export class LessonCopyUC {
 			user,
 			copyName,
 		});
+		// ---
 
 		return copyStatus;
 	}
 
+	private hasTopicCreateAndCanReadLesson(user: User, originalLesson: LessonEntity): void {
+		const contextReadWithTopicCreate = AuthorizationContextBuilder.read([Permission.TOPIC_CREATE]);
+		if (!this.authorisation.hasPermission(user, originalLesson, contextReadWithTopicCreate)) {
+			// error message is not correct, switch to authorisation.checkPermission() makse sense for me
```

Comment:
Why not change it? Seems to me like an okay change in this PR.

Reply:
Copy baut ein Status aus allen kopier Operationen auf. Ich müsste dafür das zusammen sammeln und die Client Darstellung auch noch mir im Code anschauen und evtl. anfassen.